test/metric_value_benchmark/agent_configs/statsd_config.json

```diff
@@ -1,13 +1,13 @@
 {
     "metrics": {
-        "namespace": "statsd_test",
+        "namespace": "MetricValueBenchmarkTest",
         "append_dimensions": {
             "InstanceId": "${aws:InstanceId}"
         },
         "metrics_collected": {
             "statsd": {
                 "metrics_aggregation_interval": 30,
-                "metrics_collection_interval": 5,
+                "metrics_collection_interval": 1,
```

Comment:
`metrics_collection_interval` will affect the value of `statsd` since `statsd` are using [cache](https://github.com/aws/amazon-cloudwatch-agent/blob/main/plugins/inputs/statsd/statsd.go#L75-L81) to aggregate. Therefore, `counter` in this case will have the value of 5 ([since we are sending every second with statsd](https://github.com/aws/amazon-cloudwatch-agent-test/pull/178/files#diff-68bb589fd3a05343cba4bfe377a4be5a995fd4e48a9e8ff817f3c69b5b688dceR52)) if the `metrics_collection_interval` is 5